probe/kubernetes/deployment.go

```diff
@@ -46,9 +46,13 @@ func (d *deployment) Selector() (labels.Selector, error) {
 }
 
 func (d *deployment) GetNode(probeID string) report.Node {
+	desiredReplicas := 1
+	if d.Spec.Replicas != nil {
+		desiredReplicas = int(*d.Spec.Replicas)
+	}
```

Comment:
That's what I thought. Makes writing release notes a bit tricky, though less so now that you've made it a separate commit.